### PR TITLE
Protein Databank Viewer

### PR DESCRIPTION
**Pull request by APherwani**
Updates:
- Add the molstar component
- Create custom component for gene info card
- Replace node-sass for sass because [node-sass is deprecated](https://sass-lang.com/blog/libsass-is-deprecated)

### CODE REVIEW

File: src/components/GeneInfoCard.tsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+      const response = await axios({
+        method: 'get',
+        url: `https://www.uniprot.org/uniprot/?query=${selectedItem.gene.toUpperCase()}+AND+reviewed:yes+AND+organism:9606&sort=score&columns=id&format=tab&limit=1`,
+      });
+      proteinSelected = response.data.toString();
+      console.log(response.data);
+
+      console.log('Protein selected: ' + proteinSelected);
+      let proteinId = proteinSelected.split('\n')[1];
```
Comment (suneettipirneni) [inline src/components/GeneInfoCard.tsx:34]:
```suggestion
      const [, proteinId] = proteinSelected.split('\n');
```

**Comment by joeygiordano13** [comment]
Protein renders are loading in. We need to figure out eventually how to deal with genes without alphafold models.

**Review by joeygiordano13 — APPROVED** [review]
API call to prot database works. We can transition to something else long term, but this works for now.

File: src/components/GeneInfoCard.tsx [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+  const selectedProtein = async () => {
+    try {
+      let proteinSelected = '';
+
+      const response = await axios({
+        method: 'get',
+        url: `https://www.uniprot.org/uniprot/?query=${selectedItem.gene.toUpperCase()}+AND+reviewed:yes+AND+organism:9606&sort=score&columns=id&format=tab&limit=1`,
+      });
+      proteinSelected = response.data.toString();
+      console.log(response.data);
```
Comment (suneettipirneni) [inline src/components/GeneInfoCard.tsx:31]:
```suggestion
      const proteinSelected = await axios({
        method: 'get',
        url: `https://www.uniprot.org/uniprot/?query=${selectedItem.gene.toUpperCase()}+AND+reviewed:yes+AND+organism:9606&sort=score&columns=id&format=tab&limit=1`,
      }).then(r => {
        console.log(r.data);
        return r.data.toString()
      });
```

**Review by suneettipirneni — CHANGES_REQUESTED** [review]

File: src/components/GeneInfoCard.tsx (hunk 1 above)
Comment (APherwani) [reply src/components/GeneInfoCard.tsx]: Fixed

**Comment by APherwani** [comment]
> Protein renders are loading in. We need to figure out eventually how to deal with genes without alphafold models.

As discussed with Dr. Yooseph we can just inform the client that there is no such protein to render and call it a day.


> API call to prot database works. We can transition to something else long term, but this works for now.

I think this is fine. It's just a nice touch for our app it's not a primary feature. Downloading these and adding them to an S3 bucket will mean extra plumbing on the backend.

**Comment by APherwani** [comment]
Forcing merge because I want to do some other things .

**Pull request merged**